script/release-artifact-cleanup.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+
+if (!process.env.CI) require('dotenv-safe').load()
+require('colors')
+const args = require('minimist')(process.argv.slice(2), {
+  boolean: ['tag']
+})
+const { exec, execSync } = require('child_process')
+const fail = '\u2717'.red
+const { GitProcess } = require('dugite')
+
+const GitHub = require('github')
+const pass = '\u2713'.green
+const path = require('path')
+
+const github = new GitHub()
+const gitDir = path.resolve(__dirname, '..')
+
+github.authenticate({
+  type: 'token',
+  token: process.env.ELECTRON_GITHUB_TOKEN
+})
+
+async function getCurrentBranch (gitDir) {
+  const gitArgs = ['rev-parse', '--abbrev-ref', 'HEAD']
+  const branchDetails = await GitProcess.exec(gitArgs, gitDir)
+  if (branchDetails.exitCode === 0) {
+    return branchDetails.stdout.trim()
+  } else {
+    const error = GitProcess.parseError(branchDetails.stderr)
+    console.log(`${fail} Couldn't get current branch: ${branchDetails.stderr}`, error)
+    process.exit(1)
+  }
+}
+
+async function revertBumpCommit (tag) {
+  const branch = getCurrentBranch()
+  const hashToRevert = await exec(`git log | grep 'Bump v[0-9.]*' | grep -o '\\w\\ {8,\\}' | head -n 1`)
```

Comment:
We can do this better in git:

`git log -n1 --grep "Bump v[0-9.]*" --format=format:%H`

Advantages:
 * exits after the first hit, no need to dump + grep through the entire log
 * more readable, no having to look at the output of `git --log` to guess what the `{8, \\}` means
 * could be implemented in a single `git` call, so this could be done in Dugite instead of raw exec
 * When implemented in Dugite, we can do better error checking